dracut-functions.sh

```diff
@@ -1023,3 +1023,25 @@ get_dev_module() {
     fi
     echo "$dev_drivers"
 }
+
+# Check if file is in PE format
+pe_file_format() {
+    if [[ $# -eq 1 ]]; then
+        local magic
+        magic=$(objdump -p "$1" \
+            | awk '{if ($1 == "Magic"){print strtonum("0x"$2)}}')
+        # 0x10b (PE32), 0x20b (PE32+)
+        [[ -n $magic && ($magic == 0x20b || $magic == 0x10b) ]] && return 0
```

Comment:
`magic` contains a decimal value and it's being compared to hex values.

`magic=$(printf "0x%x" $magic)` before the check?

Reply:
Thanks for this feedback.
Indeed, I thought, in the condition, hexa will be convert in decimal. I am going to update this.

Reply:
Perfect! Update done.